boards/genericSTM32F407IGT6.json

```diff
@@ -0,0 +1,55 @@
+{
+  "build": {
+    "core": "stm32",
+    "cpu": "cortex-m4",
+    "extra_flags": "-DSTM32F407xx -DSTM32F4",
+    "f_cpu": "168000000L",
+    "hwids": [
+      [
+        "0x1EAF",
+        "0x0003"
+      ],
+      [
+        "0x0483",
+        "0x3748"
+      ]
+    ],
+    "mcu": "stm32f407igt6",
+    "product_line": "STM32F407xx",
+    "variant": "Generic_F4x7Ix"
```

Comment:
What is this variant field used for? Looks like it's a leftover from the Arduino framework.

Reply:
I removed the line, but I maybe one day Arduino framework could support this board. It feels like it's just missing pin mapping ?

Reply:
Probably one day 😉